cmd/mirroring/main.go

```diff
@@ -120,13 +123,28 @@ func main() {
 	interval := flag.Duration("interval", 5*time.Minute, "Length of interval between each periodical consistency check")
 	logInfoFile := flag.String("file", logInfoFileName, "Name of the file containing initial merkle tree information")
 	once := flag.Bool("once", false, "Perform consistency check once and exit")
+	identities := flag.String("identities", "{}", "List of identities and issuers to monitor, as JSON")
```

Comment:
just curious about this, but what about removing JSON and just requesting it on a line by line separated

SUBJECTA ISSUERA.0 ISSUERA.1
SUBJECTB ISSUERB
SUBJECTC

Reply:
i like it! went with that